definitions/divorce/json/CaseEvent.json

```diff
@@ -2422,5 +2422,17 @@
     "PostConditionState": "*",
     "SecurityClassification": "PUBLIC",
     "ShowSummary": "Yes"
+  },
+  {
+    "LiveFrom": "23/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "solApplyForDA",
+    "Name": "Solicitior applies for DA",
```

Comment:
Can you please change this to "Apply for Decree Absolute"? 
That is the name of the event visible to the solicitor in CCD.